e2e/playwright/text-to-cad-tests.spec.ts

```diff
@@ -689,3 +691,749 @@ test(
+ * from real API requests which are copied and pasted below
+ *
+ * Gotcha: The exports OBJ etc... are not in the output they are massive.
+ *
+ * Gotcha: Yes, the 3D render preview will be broken because the exported models
+ * are not included. These tests do not care about this.
+ *
+ */
+test.describe('Mocked Text-to-CAD API tests', { tag: ['@skipWin'] }, () => {
+  async function mockPageTextToCAD(page: Page) {
+    await page.route(
+      'https://api.dev.zoo.dev/ai/text-to-cad/glb?kcl=true',
+      async (route) => {
+        const json = {
+          id: 'bfc0ffc0-46c6-48bc-841e-1dc08f3428ce',
+          created_at: '2025-04-24T16:50:48.857892376Z',
+          user_id: '85de7740-3e38-4e86-abb5-e5afbb8a2183',
+          status: 'queued',
+          updated_at: '2025-04-24T16:50:48.857892376Z',
+          prompt: '1x1x1 cube',
+          output_format: 'glb',
+          model_version: '',
+          kcl_version: '0.2.63',
+          model: 'kcl',
+          feedback: null,
+          mocked: true,
+        }
+        await route.fulfill({ json })
+      }
+    )
+    await page.route(
+      'https://api.dev.zoo.dev/user/text-to-cad/*',
+      async (route) => {
+        const json = {
+          mocked: true,
+          id: '6ecbb863-2766-47f0-95cb-7122ad7560ce',
+          created_at: '2025-04-24T16:52:00.360Z',
+          started_at: '2025-04-24T16:52:00.360Z',
+          completed_at: '2025-04-24T16:52:00.363Z',
+          user_id: '85de7740-3e38-4e86-abb5-e5afbb8a2183',
+          status: 'completed',
+          updated_at: '2025-04-24T16:52:00.360Z',
+          prompt: '2x2x2 cube',
+          outputs: {},
+          output_format: 'step',
+          model_version: '',
+          kcl_version: '0.2.63',
+          model: 'kcl',
+          feedback: null,
+          code: '/*\nGenerated by Text-to-CAD:\n2x2x2 cube\n*/\n@settings(defaultLengthUnit = mm)\n\n// Define the dimensions of the cube\ncubeSide = 2\n\n// Start a sketch on the XY plane\ncubeSketch = startSketchOn(XY)\n  |> startProfileAt([0, 0], %)\n  |> line(end = [cubeSide, 0])\n  |> line(end = [0, cubeSide])\n  |> line(end = [-cubeSide, 0])\n  |> close()\n\n// Extrude the sketch to create a 3D cube\ncube = extrude(cubeSketch, length = cubeSide)',
+        }
+        await route.fulfill({ json })
+      }
+    )
+  }
+
+  test(
+    'Home Page -> Text To CAD -> New Project -> Stay in home page -> Reject -> Project should be deleted',
+    { tag: '@electron' },
+    async ({ context, page }, testInfo) => {
+      const projectName = 'my-project-name'
+      const prompt = '2x2x2 cube'
+      await mockPageTextToCAD(page)
+
+      // open commands
+      await page.getByTestId('command-bar-open-button').click()
+
+      // search Text To CAD
+      await page.keyboard.type('Text To CAD')
+      await page.keyboard.press('Enter')
+
+      // new project
+      await page.keyboard.type('New project')
+      await page.keyboard.press('Enter')
```

Comment:
These tests should make use of the `cmdBar` fixture in the future, like this example here

https://github.com/KittyCAD/modeling-app/blob/main/e2e/playwright/point-click-assemblies.spec.ts#L33-L38